### PR TITLE
FDG-2348 Add stagename to jobname

### PR DESCRIPTION
**Pull request by bhatiakartik**

### CODE REVIEW

**Review by saurabh-jha — APPROVED** [review]
LGTM

**Review by vishwa-nath — APPROVED** [review]
LGTM

File: ql/src/java/org/apache/hadoop/hive/ql/Driver.java [hunk 1]
```diff
@@ -2156,9 +2158,12 @@ private TaskRunner launchTask(Task<? extends Serializable> tsk, String queryId,
       SessionState.get().getHiveHistory().startTask(queryId, tsk, tsk.getClass().getName());
     }
     if (tsk.isMapRedTask() && !(tsk instanceof ConditionalTask)) {
+      //ENTRY POINT FOR SETTING JOB NAME . HERE WE SHOULD SET OUR REQUIRED JOB NAME
       if (noName) {
         conf.set(MRJobConfig.JOB_NAME, jobname + "(" + tsk.getId() + ")");
       }
+      FDPPropertySetter.setJobName(FDPAuth.getInstance(),
```
Comment (amrk7s) [inline ql/src/java/org/apache/hadoop/hive/ql/Driver.java:2165]: Would whitelisted hosts also need to follow this.

**Review by amrk7s — APPROVED** [review]
LGTM

**Pull request closed without merging**